portfolio/src/main/webapp/index.html

```diff
@@ -5,35 +5,57 @@
     <title>My Portfolio | Peter Wu</title>
     <link rel="stylesheet" href="constants.css" />
     <link rel="stylesheet" href="style.css" />
-    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
-    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="favicon.ico" type="image/x-icon" />
     <script async src="script.js"></script>
   </head>
   <body>
     <div class="NavbarWrap">
-      <a href="#" id="logoButton">
-        <img src="/assets/images/logo.png" alt="Peter Wu logo" width="72px" />
+      <a href="#" id="LogoButton">
```

Comment:
It is good to keep the naming consistent, so I just put it as FYI: Google CSS style guide recommends to use all lowercases in html and separate words with dashes.

Reply:
Done - just refactored to fit the style guidelines. It seems like interns don't have access to the internal version of the style guide, so I've been referring to the one on google.github.io.